.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: build
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
Hm, do we want this here? This will trigger the build on `release/2.8` on push to `main`, right? 🤔

Reply:
We can safely remove it yes

Reply:
I'd say let's remove everything that can confuse the noobs like myself 😅

Reply:
Done